apps/docs/stories/components/payment-form-example.ts

```diff
@@ -1,12 +1,26 @@
-import { codeExampleHead } from '../utils';
+import { codeExampleHead } from "../utils";
 
 export default `<!DOCTYPE html>
 <html dir="ltr" lang="en">
 
-${codeExampleHead('justifi-payment-form')}
+${codeExampleHead("justifi-payment-form")}
+
+/*
+* Props definition:
+*
+* "client-id is you JustiFi API Credential Client Id
```

Comment:
Should be 
client-id is your JustiFi API Credential Client Id
`"` in beginning of line not needed and missing `r` in `your`